frontend/packages/console-app/src/components/user-preferences/lightspeed/useHideLightspeed.ts

```diff
@@ -0,0 +1,12 @@
+import { useUserSettings } from '@console/shared';
+
+const PREFERRED_LIGHTSPEED_USER_SETTING_KEY: string = 'console.hideLightspeedButton';
+
+export const useHideLightspeed = (): [boolean, boolean] => {
```

Comment:
@kyoto, FYI.  You'll want to duplicate and make use of this hook in the plugin so the user preference works there, too.